webapp/components/NewsItem.vue

```diff
@@ -1,36 +1,56 @@
 <template>
   <div class="news-item">
-    <h2><span class="news-message">{{ newsItem.title }}</span> ({{ newsItem.voteCount }})</h2>
+    <h2><span class="news-message">{{ newsItem.title }}</span> ({{ newsItem.votes }})</h2>
     <span>
-      <button @click="upvote">upvote</button>
-      <button @click="downvote">downvote</button>
-      <button @click="remove" class="remove-button">remove</button>
+      <button @click="upvote" v-if="loggedIn">upvote</button>
+      <button @click="downvote" v-if="loggedIn">downvote</button>
+      <button @click="remove" v-if="loggedIn && parseInt(newsItem.author.id) === currentUser" class="remove-button">remove</button>
```

Comment:
⭐ For a delete and edit button that is only visible to the author of the post.

Reply:
`currentUser` is a number? Improve naming, please.